internal/common/common.go

```diff
@@ -109,6 +109,20 @@ func ReadLinesOffsetN(filename string, offset uint, n int) ([]string, error) {
 	return ret, nil
 }
 
+// Reads the contents of a directory
+func ReadDirectory(path string) ([]string, error) {
```

Comment:
nit: ListDirectory seems clearer (to me) what this will return. ReadDirectory seems like you might be the contents of all files in the directory